### ISSUE TITLE
real time

### ISSUE THREAD

**Issue by luantunez** [WillBrennan/SemanticSegmentation#14]
Hello and thank you for sharing your work!
I do not understand entirely how does the real time model for skin segmentation work.
Do you have some instructions regarding, the input, the output, and the purpose?
Thank you, Lucia

**Comment by hawkeye-will** [comment]
Hi there; 
The steps for recreating the skin segmentation code is in the readme. First start by [creating your enviroment](https://github.com/WillBrennan/SemanticSegmentation#pre-trained-segmentation-projects), then follow the section on [running the pretrained projects](https://github.com/WillBrennan/SemanticSegmentation#pre-trained-segmentation-projects). 

This code was created as a machine-learning version of the original [SkinDetector](https://github.com/WillBrennan/SkinDetector) project. That project was created to show how multi-colour space thresholding is used in traditional image processing to create segmentation masks.

**Comment by WillBrennan** [comment]
Hi there; 
The steps for recreating the skin segmentation code is in the readme. First start by [creating your enviroment](https://github.com/WillBrennan/SemanticSegmentation#pre-trained-segmentation-projects), then follow the section on [running the pretrained projects](https://github.com/WillBrennan/SemanticSegmentation#pre-trained-segmentation-projects). 

This code was created as a machine-learning version of the original [SkinDetector](https://github.com/WillBrennan/SkinDetector) project. That project was created to show how multi-colour space thresholding is used in traditional image processing to create segmentation masks.

**Comment by luantunez** [comment]
Thank you very much for your quick response. I have run your code for an image with the FCNResNet101 model. It works great!
But what I do not fully understand is the BiSeNetV2, for the real-time models. What does it mean being real time? Does it also process an image, or a video instead? What is the difference between both models?
Thank you, Lucia

**Comment by WillBrennan** [comment]
On a HD image; BiSeNet can run at 60fps while ResNet101 can barely handle 1fps on a 1070. This code is only written for single images, but you can run BiSeNet in "real-time"

**Comment by luantunez** [comment]
Perfect! Thank you!
